includes/class-scheduler.php

```diff
@@ -22,6 +25,21 @@ public static function init() {
 		\add_action( 'activitypub_cleanup_followers', array( self::class, 'cleanup_followers' ) );
 
 		\add_action( 'admin_init', array( self::class, 'schedule_migration' ) );
+
+		// profile updates for blog options
+		if ( ! is_user_type_disabled( 'blog' ) ) {
+			\add_action( 'update_option_site_icon', array( self::class, 'blog_user_update' ) );
+			\add_action( 'update_option_blogdescription', array( self::class, 'blog_user_update' ) );
+			\add_action( 'update_option_blogname', array( self::class, 'blog_user_update' ) );
+			\add_filter( 'pre_set_theme_mod_custom_logo', array( self::class, 'blog_user_update' ) );
+			\add_filter( 'pre_set_theme_mod_header_image', array( self::class, 'blog_user_update' ) );
+		}
+
+		// profile updates for user options
+		if ( ! is_user_type_disabled( 'user' ) ) {
+			\add_action( 'updated_user_meta', array( self::class, 'user_update' ), 10, 3 );
+			// @todo figure out a feasible way of updating the header image since it's not unique to any user.
```

Comment:
@mattwiebe @pfefferle I have a Profile Header code stub, which could live inside this plugin, but also makes sense as an independent block for Block Theme use. Any thoughts?

Reply:
@mediaformat sounds interesting! Do you have a public repo with the code?